### DISCUSSION TITLE
Can I reduce the frequency of Dagster calling GetServerId?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18085] [Q&A]
Hi there...I'm trying to implement datadog with my dagster pipeline and am seeing frequent trace capture from the endpoint api.DagsterAPI GetServerId.  It is called quite frequently so I"m trying to figure out its purpose so I can reduce the frequency, but am having trouble finding the documenation.  My implementation is running Dagster in ECS /grpc. Can anyone provide any insight?  Thanks!

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1700152858194909?thread_ts=1700152858.194909&cid=C01U954MEER).*

**Comment by prha** [comment]
GetServerId serves as a heartbeat check from the Dagster webserver and daemon processes to the individual code location servers.  It powers the health checks for the different code locations and checks for changes to reload the definitions in the frontend.

I believe this gets called every second.

**Reply by hai-benwilson** [reply]
Thanks @prha that makes sense.  Do you know if it's possible to change the rate at which it's called?

**Reply by prha** [reply]
I don't think this is configurable a.t.m.  Is it possible to blacklist the endpoint for trace purposes?  Is this causing issues on your end?
